src/Message.php

```diff
@@ -4,14 +4,23 @@
 
 class Message
 {
-    /** @var string */
+    /** @var string|array */
     public $message;
 
     /** @var string */
     public $class;
 
-    public function __construct(string $message, $class)
+    public function __construct($message, $class)
     {
+        if (is_array($message)) {
+            $concat = collect($message)->map(function ($msg) {
+                return '<li>'.$msg.'</li>';
```

Comment:
Instead of hardcoding html here we could do the rendering in a view. So if people want to have different markup they can customize it.